packages/story-editor/src/components/header/buttons/publish.js

```diff
@@ -109,44 +99,36 @@ function PublishButton({ forceIsSaving }) {
   }, [refreshPostEditURL, saveStory, hasFutureDate, title, canPublish]);
 
   const handlePublish = useCallback(() => {
-    if (
-      canPublish &&
-      (shouldReviewDialogBeSeen || isUpdatedPublishModalEnabled)
-    ) {
+    updateToAllowPriorityIssues();
+    if (shouldReviewDialogBeSeen || isUpdatedPublishModalEnabled) {
       setShowDialog(true);
       return;
     }
 
     publish();
   }, [
+    updateToAllowPriorityIssues,
     shouldReviewDialogBeSeen,
     isUpdatedPublishModalEnabled,
-    canPublish,
```

Comment:
🤔 is this not necessary anymore?

Reply:
nope because `isUpdatedPublishModalEnabled` is enabled by default right now so when we clean out the feature flag all this is going to end up being is 
```
const handlePublish = useCallback(() => {
 updateToAllowPriorityIssues();
setShowDialog(true);
}, [updateToAllowPriorityIssues])
```